modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/LogicalRelImplementor.java

```diff
@@ -744,7 +744,10 @@ public Node<RowT> visit(IgniteReduceHashAggregate rel) {
         RelDataType rowType = rel.getRowType();
 
         Supplier<List<AccumulatorWrapper<RowT>>> accFactory = expressionFactory.accumulatorsFactory(
-                type, rel.getAggregateCalls(), null);
```

Comment:
`IgniteReduceHashAggregate / IgniteReduceSortAggregate` should have the same accFactory setup.